snippets/backticks.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+``;
```

Comment:
```suggestion
`ls -lsa`;
```

Reply:
I thought about that, but the `exec` example is empty too and I checked, the violation is reported even if the backticks  have no content.

Reply:
please update exec as well :)